crates/red_knot_test/README.md

```diff
@@ -283,7 +283,7 @@ cargo test -p red_knot_python_semantic -- mdtest__
 Alternatively, you can use the `mdtest.py` runner which has a watch mode that will re-run corresponding tests when Markdown files change, and recompile automatically when Rust code changes:
 
 ```bash
-uv -q run crates/red_knot_python_semantic/mdtest.py
+uv run crates/red_knot_python_semantic/mdtest.py
```

Comment:
Drive-by fix, `-q` is not needed anymore to silence the "Reading inline script metadata from …" message since https://github.com/astral-sh/uv/pull/10588